app/Auth/Access/SocialAuthService.php

```diff
@@ -52,7 +52,12 @@ public function startLogIn($socialDriver)
     public function startRegister($socialDriver)
     {
         $driver = $this->validateDriver($socialDriver);
-        return $this->socialite->driver($driver)->redirect();
+        if ($socialDriver == 'google') {
+            return $this->socialite->driver($driver)->with(['prompt' => 'select_account'])->redirect();
+        }
+        else {
+            return $this->socialite->driver($driver)->redirect();
```

Comment:
The else here isn't really required. Removing the else would make it clear what the default action is.